example/test/readme_test.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:test/test.dart';
+
+void main() {
+  _expect('README.md');
+  _expect('pubspec.yaml');
+}
+
+void _expect(String fileName) {
+  test(fileName, () {
+    final file = new File(fileName);
+    expect(file.readAsStringSync(), contains(_pubspecContent));
```

Comment:
what is the value here? Making sure that versions are kept up to date? Should you read the pubspecs instead and use the versions from there?

Reply:
Then I have to read that file and figure out which lines I want. This seemed easier.